multigeiger/loraWan.cpp

```diff
@@ -1,7 +1,26 @@
-#ifndef LORAWAN_H
-#define LORAWAN_H
```

Comment:
such stuff is usually found in the `.h` file (thus the LORAWAN_H name) - why was it here?